### PR TITLE
Add japanese translation for FAQ page

### PR DESCRIPTION
**Pull request by dowdiness**
Hi. I finished to translate FAQ page into Japanese. Please take a look to this.

### CODE REVIEW

**Comment by bluebill1049** [comment]
thank you so much! we are going live with all of these soon.

**Comment by bluebill1049** [comment]
@kotarella1110 shall we merge this PR first?

File: src/data/faq.tsx [hunk 1]
```diff
@@ -711,196 +711,177 @@ export default {
...
-              definitely mix classes and function components with Hooks in a
-              single tree. Whether a component is a class or a function that
-              uses Hooks is an implementation detail of that component. In the
-              longer term, we expect Hooks to be the primary way people write
-              React components.
+            クラスコンポーネント内でのHooksの使用は出来ませんが、クラスコンポーネントと
+            Hooksを使用した関数コンポーネントを単一のDOM Tree内で混在させることは可能です。
+            クラスコンポーネントかHooksを使用した関数コンポーネントのどちらを利用するかは、
+            そのコンポーネントの実装の詳細に過ぎません。長い目で見れば、
+            HooksはReaceのコンポーネントを書く際の最初の選択肢になっていくだろうと私達は予想しています。
```
Comment (kotarella1110) [inline src/data/faq.tsx]: typo: Reace => React

Comment (kotarella1110) [inline src/data/faq.tsx]: typo: Reace => React

File: src/data/faq.tsx [hunk 2]
```diff
@@ -711,196 +711,177 @@ export default {
...
               rel="noopener noreferrer"
               target="_blank"
             >
-              Uncontrolled Component
+              非制御コンポーネント
             </a>
-            , which gives you the ability to build an accessible custom form
-            easily.
+            によって出来ており、アクセス可能なカスタムフォームを簡単に構築できます。
           </p>
```
Comment (kotarella1110) [inline src/data/faq.tsx]: This expression better "非制御コンポーネントに基づいているため" or "非制御コンポーネントに基づいて構築されているため" than "非制御コンポーネントによって出来ており".

File: src/data/faq.tsx [hunk 3]
```diff
@@ -711,196 +711,177 @@ export default {
...
       },
       {
-        title: "Does it work with Class Components?",
+        title: "クラスコンポーネントでも使えますか？",
         description: (
           <>
             <p>
-              No, not out of box. but you can build a wrapper around it and use
-              in your Class Component.
+              そのままでは動作しませんが、React Hook Formを包むラッパーを作成してクラスコンポーネント内で使用することは可能です。
```
Comment (kotarella1110) [inline src/data/faq.tsx]: The word "it" meaning is "Class Component".
This expression bette  for "クラスコンポーネントを包むラッパーを作成して使用することは可能です。".

**Comment by dowdiness** [comment]
Hi @kotarella1110. Thank you for reviewing my translation and correct some mistakes.  I believe these suggestions make this translation better.

**Comment by dowdiness** [comment]
@bluebill1049 @kotarella1110  I think this PR is ready to merge.

**Comment by bluebill1049** [comment]
thank you @dowdiness

**Comment by bluebill1049** [comment]
we are ready to go live with the Japanese version :)

**Comment by bluebill1049** [comment]
Do you have a terrier account? @dowdiness so I can tag your name when we go live.

**Comment by dowdiness** [comment]
I don't know what terrier account means... Anyway this is my twitter account. https://twitter.com/kojii0817 You can tag my twitter or github account.

**Comment by bluebill1049** [comment]
oops i meant twitter

**Pull request merged**